vault/metadata.csv

```diff
@@ -37,33 +37,508 @@ vault.process.resident_memory.bytes,gauge,,,,Resident memory size in bytes.,0,va
+vault.database.initialize.count,gauge,,millisecond,,Time taken to initialize a database secret engine across all database secrets engines count,0,vault,
+vault.database.initialize.quantile,gauge,,millisecond,,Time taken to initialize a database secret engine across all database secrets engines quantile,0,vault,
+vault.database.initialize.sum,gauge,,millisecond,,Time taken to initialize a database secret engine across all database secrets engines sum,0,vault,
+vault.database.initialize.error,count,,error,,Number of databases secrets engine initialization operation errors across all database secrets engines,0,vault,
+vault.database.close.count,gauge,,millisecond,,Time taken to close a database secret engine across all database secrets engines count,0,vault,
+vault.database.close.quantile,gauge,,millisecond,,Time taken to close a database secret engine across all database secrets engines quantile,0,vault,
+vault.database.close.sum,gauge,,millisecond,,Time taken to close a database secret engine across all database secrets engines sum,0,vault,
+vault.database.close.error,count,,error,,Number of database secrets engine close operation errors across all database secrets engines,0,vault,
+vault.database.createUser.count,gauge,,millisecond,,Time taken to create a user across all database secrets engine count,0,vault,
+vault.database.createUser.quantile,gauge,,millisecond,,Time taken to create a user across all database secrets engine quantile,0,vault,
+vault.database.createUser.sum,gauge,,millisecond,,Time taken to create a user across all database secrets engine sum,0,vault,
+vault.database.createUser.error,count,,error,,Number of user creation operation errors across all database secrets engines,0,vault,
+vault.database.renewUser.count,gauge,,millisecond,,Time taken to renew a user across all database secrets engines count,0,vault,
+vault.database.renewUser.quantile,gauge,,millisecond,,Time taken to renew a user across all database secrets engines quantile,0,vault,
+vault.database.renewUser.sum,gauge,,millisecond,,Time taken to renew a user across all database secrets engines sum,0,vault,
+vault.database.renewUser.error,count,,error,,Number of user renewal operation errors across all database secrets engines,0,vault,
+vault.revokeUser.count,gauge,,millisecond,,Time taken to revoke a user across all database secrets engines count,0,vault,
+vault.revokeUser.quantile,gauge,,millisecond,,Time taken to revoke a user across all database secrets engines quantile,0,vault,
+vault.revokeUser.sum,gauge,,millisecond,,Time taken to revoke a user across all database secrets engines sum,0,vault,
+vault.revokeUser.error,count,,error,,Number of user revocation operation errors across all database secrets engines,0,vault,
+vault.vault.azure.put.count,gauge,,millisecond,,Duration of a PUT operation against Azure storage backend count,0,vault,
```

Comment:
`vault.vault` ?

Reply:
For the metric `vault.vault.azure.put`, the first `vault` is prefixed by the namespace and the second is the metric name (`vault_azure_put`). However the metric `vault.revokeUser.error` is prefixed with the namespace and the default metric name is `database_RevokeUser_error`.

Some Vault metrics start with `vault` and others don't (like replication and secrets engine metrics). We decided to keep `vault.vault` so that future metrics added do not conflict with the ones already collected and consistent with Vault's documentation.